Draft of revisions to results section (#39)
greenelab/annorxiver_manuscript: content/04.results.md

The average sentence length, fraction of pronouns, and the use of the passive voice were all more similar between bioRxiv and PMC than they were to NYTAC (Table {@tbl:corpora_stats}).
The Kullback–Leibler divergence measures the extent to which two distributions, but not the specific entities that comprise those distributions, differ.
The distribution of term frequencies in bioRxiv and PMC were low, especially among the top few hundred tokens (Figure {@fig:corpora_comparison_panels}A).
Differences began to emerge over more terms, but remained much lower than when the biomedical corpora are compared with NYTAC.

Examining the frequencies of individual terms revealed differences between the biomedical corpora.
Previous work examining author-selected categories has reported that fields appear to have preprinted unevenly, with certain life sciences research fields having more uptake than others [@doi:10.7554/eLife.45133].
However, it was possible that authors simply selected certain fields preferentially but that the content was similar to the broader corpus of life sciences text.
We directly examined this by comparing term frequencies between bioRxiv and PMC.

> We found that among the terms that differed the most <!-- HOW DID YOU PICK THE TERMS FOR THE PLOT? I THINK IT WAS A COMBO OF FREQUENCY AND OR --> were many associated with fields.

Reviewer: ```suggestion
We found that among the terms that differed the most many were associated with certain life sciences research fields.
```

The horizontal lines represent the overall proportion of preprints that are were published as of the time of the annotation snapshot.
](https://raw.githubusercontent.com/danich1/annorxiver/f9d8861e3d60afa878c5d0088b7502adae15ebe3/figure_generation/output/figure_three_panels.svg){#fig:preprint_links_panels width="100%"}

Based on this finding, we analyzed preprint-publication pairs that were close in document space but not annotated as such.
We separated these pairs into four quantiles with the first three based on the distribution of preprint-publication distances and the fourth going from the 75th percentile in the preprint-publication pair space to the smallest value observed for the preprint-random set.
We then selected 50 preprint-publication pairs from each of these sets and shuffled them to create a random list of 200 possible pairs.
Two scientists then examined the pairs in these randomized lists determining if an preprint-publication pair represented the peer reviewed publication of the primary content described in the preprint.

> Across the entire list we found a high inter-rater reliability of with a Cohen's Kappa [@doi:10.1177/001316446002000104] of 0.92. <!-- CHECK THIS - IT WAS PRESENTED AS A PERCENT BUT I'M PRETTY SURE IT SHOULDN'T BE IF IT'S COHEN'S KAPPA -->

Reviewer: ```suggestion
Across the entire list we found a high inter-rater reliability of with a Cohen's Kappa [@doi:10.1177/001316446002000104] of 0.92.
```

Reviewer: Validating that this value should not be a percentage.

The y-axis shows the number of days that elapsed between when the first version of a preprint was posted on bioRxiv and the date at which the peer reviewed publication appeared.
The density of observations are depicted with the violin plot with an embedded boxplot.
C. Author-selected categories were associated with modest differences in the time to publish.
Categories are shown on the y-axis.
The x-axis shows the median time-to-publish for each category.
Error bars represent 95% confidence intervals for each preprint category's median time to publication.
](https://raw.githubusercontent.com/danich1/annorxiver/232ab6f9f29f283070ed08b88a421b3bedaf5cbd/figure_generation/output/figure_four_panels.svg){#fig:publication_delay_panels width="100%"}

The process of peer review includes a number of steps which take variable amounts of time [@doi:10.1002/nop2.51].
Comparing bioRxiv preprints with their corresponding published version provides an opportunity to better understand peer review and publishing.
We examined how long it took to publish preprints in their peer reviewed form for each author-selected category (Figure {@fig:publication_delay_panels}C).

> We treated examined this using the Kaplan-Meier estimator, considering the endpoint to be publication.

Reviewer: ```suggestion
We considered a preprint's endpoint to be publication and used the Kaplan-Meier estimator to estimate how much time has elapsed.
```

> Of the most abundant preprint categories microbiology was the fastest to publish (point estimate, confidence intervals) and genomics was the slowest (point estimate, confidence intervals). <!--TODO: fill in the point estimates and confidence intervals-->

Reviewer: ```suggestion
Of the most abundant preprint categories microbiology was the fastest to publish (140 days, (137, 145 days) [95% CI]) and genomics was the slowest (190 days, (185, 195 days) [95% CI]).
```

Though we did observe category-specific differences, these differences were generally modest.

> One exception was the Scientific Communication and Education category, which took substantially longer to be peer reviewed and published (point estimate, confidence interval).

Reviewer: ```suggestion
One exception was the Scientific Communication and Education category, which took substantially longer to be peer reviewed and published (373 days, (373, 398 days) [95% CI]).
```

### Preprints with similar document embeddings share publication venues

We hypothesized that document embedding might capture information associated with the eventual publication venue.
We used the annotated preprint-publication pairs as a fully held out test set.
We embedded all PMC papers except those annotated to have a corresponding preprint into the bioRxiv embedding space.
For journals, we averaged the document representation of all papers in the journal to produce a journal centroid.
We used a k-nearest neighbor classifier trained on the remaining PMC documents to identify the most similar published papers and assessed performance - the extent to which the journal a paper appeared in the returned list of ten neighbors - through cross validation and the held out test set (Supplemental Figure {@fig:knn_auc}).
We found an enrichment of papers published within the same journal among the neighbors of the target paper.

We developed an online app that returns the nearest published papers and journals that are nearest to a preprint.

> Users supply DOIs from _bioRxiv_ or _medRxiv_. <!-- we need to be consistent in how these are represented - should they always be italicized? it feels like a journal name would be - perhaps see what the bioRxiv folks did in their own paper. -->

Reviewer: ```suggestion
Users supply DOIs from bioRxiv or medRxiv. 
```

Reviewer: bioRxiv wasn't italicized in the bioRxiv folks' paper but journals like PLOS genetics and Proceedings of the Royal Society were. Since bioRxiv isn't a journal, I don't believe we need to italicize it.

The application downloads the article from the preprint repository, converts the PDF to text, calculates a document embedding score, and returns the ten papers and journals with the most similar representations in the embedding space (Figure {@fig:journal_rec_workflow}).
We also sought to display the position of each preprint in the overall landscape.
We required an embedding that could be rapidly calculated for new documents, so we used the Sparse Autoencoder for Unsupervised Clustering, Imputation, and Embedding (SAUCIE) approach previously described for the analysis of single-cell gene expression data [@doi:10.1038/s41592-019-0576-7].

> The user-requested preprint's location in this space is then displayed.
> Users can select regions to identify the terms most associated with those regions.

Reviewer: ```suggestion
The user-requested preprint's location in this space is then displayed and users can select regions to identify the terms most associated with those regions.
```

Users can also explore the terms associated with the top 50 PCs derived from the document embeddings and those PCs vary across the document landscape.

![

> The journal recommender app workflow allows users to examine where an individual preprint falls in the overall document landscape.

Reviewer: I thought we were shying away from calling it a "recommender" app and moving towards calling it a similarity search app.
```suggestion
The journal similarity search app workflow allows users to examine where an individual preprint falls in the overall document landscape.
```

> A. Starting with the home screen, users can paste in a _bioRxiv_ or _medRxiv_ DOI, which sends a request to bioRxiv or medRxiv.

Reviewer: ```suggestion
A. Starting with the home screen, users can paste in a bioRxiv or medRxiv DOI, which sends a request to bioRxiv or medRxiv.

```
